frontend-react/src/pages/product/legacy-page-content/WhereWereLive.tsx

```diff
@@ -26,8 +26,21 @@ export const WhereWereLive = () => {
                 src={usamapsvg}
                 alt="Map of states using ReportStream"
             />
-            ReportStream has established connections to send and report public
-            health data for each of the states and territories listed here.
+            <p className="font-body-2xs text-gray-50">
+                <a
```

Comment:
VERY superficial point completely out of curiosity, but do we have a general policy of opening external links in a new tab or keeping it on the same tab?

Reply:
Great question! So as of `2.11` they removed their official guidance on the matter due to confusion from user research over the icon itself meaning "share", potentially. However, no guidance on the page now indicates _not_ to do it, which tells me it's more of an "us" call!

https://designsystem.digital.gov/components/link/ (scroll to bottom)

<img width="982" alt="Screen Shot 2022-12-19 at 5 19 00 PM" src="https://user-images.githubusercontent.com/4022304/208538001-ad09dc6e-85f2-4caa-87e2-111112ae077a.png">

Reply:
Makes sense!  Thanks for following up!